includes/Modules/Analytics.php

```diff
@@ -1236,6 +1326,17 @@ protected function setup_settings() {
 		return new Settings( $this->options );
 	}
 
+	/**
+	 * Gets the provisioning redirect URI that listens for the Terms of Service redirect.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string Provisioning redirect URI.
+	 */
+	private function get_provisioning_redirect_uri() {
+		return add_query_arg( 'gatoscallback', '1', untrailingslashit( home_url() ) );
```

Comment:
@felixarntz should we be introducing new boolean query params to listen for or does the API require this param specifically? I feel like we could get by just as well with a single parameter such as `googlesitekit-callback={callback-id}`. This would likely cover most of our cases, since I don't think we ever need to run multiple callbacks of this type in the same request and avoid potential conflicts with other plugins in the future.

Reply:
I agree with what you're saying, but we've already defined this in several places, on the authentication service too. It's also in line with our other callback URL, which isn't really a point, but I think it's okay to use the parameter like this.

It's not a big deal to me, the only reason I'd say we shouldn't change it despite your argument is because it would require us to make changes across several places.